boefjes/boefjes/plugins/kat_fierce/fierce.py

```diff
@@ -41,7 +42,7 @@ def print_subdomain_result(url, ip, nearby=None):
 
 
 def unvisited_closure():
-    visited = set()
+    visited: set = set()
```

Comment:
This isn't necessary right?

Reply:
It is because mypy can't figure out the type of the members from this assignment because it is an empty set. If you remove it you get the error:

```
boefjes/boefjes/plugins/kat_fierce/fierce.py:45: error: Need type annotation for "visited" (hint: "visited: Set[<type>] = ...")  [var-annotated]
```

Reply:
Ah right, good to know 👍